### ISSUE TITLE
Not working on Japanese Steam version

### ISSUE THREAD

**Issue by Quenser** [Wulf2k/DaS-PC-MPChan#126]
Hello,

the mod doesn't seem to be compatible with the Japanese version of Dark Souls that got added to Steam in January. Is there any possibility of you fixing it or releasing a branch for the Japanese version?

**Comment by Wulf2k** [comment]
It is possible, but would need every single hard-coded address to be updated.

This would be a bit of a project that I don't currently have time for but would like to do eventually.

**Comment by Chronial** [comment]
Do we know whether the japanese version is multiplayer-compatible with the english version?

**Comment by Wulf2k** [comment]
I could swear that I've heard of people using DSCM to manually add JP players, but I haven't confirmed it myself.

**Comment by Chronial** [comment]
Do we know whether the japanese version is multiplayer-compatible with the english version?

**Comment by Chronial** [comment]
@Quenser the one of you running the english version can just manually connect to the one without the english version via DSCM. The other person does not need DSCM for this to work.

**Comment by Quenser** [comment]
I can certainly try that, however it would still leave my friend and all other Japanese version players (probably all 10 of them) without properly working online. I can see why this is much more work than it is worth. I was hoping it would be an easy fix, it can't be helped.

**Comment by Chronial** [comment]
Sry, you misunderstand my purpose: It matters a lot for this task whether the english version and japanese version can connect to each other.
If they can't, they create two completely separate networks. Then we would also have to maintain two separate dscm-nets.

If they can connect to each other, it's definitely easier to adapt DSCM to the japanese version.

**Comment by Quenser** [comment]
No, I understood what you were asking of me, I'll try it as soon as I can.

**Comment by Quenser** [comment]
I just got to try it and it works with no issue. Can connect to Japanese version.
